ansible_collections/arista/avd/roles/eos_validate_state/README.md

```diff
@@ -95,10 +95,23 @@ validate_state_markdown_flavor: "default"
 
 # Allow different manufacturers
 accepted_xcvr_manufacturers: "{{ validation_role.xcvr_own_manufacturers | arista.avd.default(['Arastra, Inc.', 'Arista Networks']) }}"
+
```

Comment:
We rely on `fabric_name` being set to the same as a group covering all the devices.
a. We add a default value `fabric_name: all` to the role defaults
b. We add documentation for this variable (just copy the section from eos_designs on `fabric_name`)